.travis.yml

```diff
@@ -39,4 +38,4 @@ notifications:
   webhooks:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
-    on_start: false     # default: false
```

Comment:
What's this for? :)

Reply:
Oh, OK.

Reply:
> What's this for? :)

Well `false` seems to be an invalid value, especially because it is not the default (any more)...